pkg/volume/csi/csi_plugin_test.go

```diff
@@ -243,36 +243,6 @@ func TestPluginNewUnmounter(t *testing.T) {
 
 }
 
-func TestValidateDriverName(t *testing.T) {
```

Comment:
move test to keep testcases against the regex?

Reply:
fixed